app/src/main/res/values-es/strings.xml

```diff
@@ -31,7 +31,8 @@
     <string name="dices_d20">D20</string>
     <string name="dices_d12">D12</string>
     <string name="dices_slider_hint">Número de dados: %d</string>
-    <string name="dices_d_d">D%d</string>
+    <string name="dices_inc_slider_hint">Agregar para contar: %d</string>
```

Comment:
Do you speak all these languages or are these translations auto generated? I prefer to keep autogenerated translations out of this project and wait for other contributors to add missing translations.

Reply:
I will remove the translations